dev/org.eclipse.codewind.openapi.ui.test/src/org/eclipse/codewind/openapi/ui/test/BaseTestCase.java

```diff
@@ -56,18 +122,19 @@ protected void createGeneralProject(String projectName) {
 		assertNotNull("Project creation", this.testProject);
 		assertEquals("Test project name", projectName, this.testProject.getName());
 	}
-	
+
 	/**
 	 * 
-	 * @param sourceFile - source OpenAPI definition in the test plugin resources folder
-	 * @param destinationFile - path of target OpenAPI definition in the test project but excludes 
-	 *                          the project name
+	 * @param sourceFile      - source OpenAPI definition in the test plugin
```

Comment:
I'm using the Eclipse [built-in] preferences with a maximum column width of 120.  I think the built-in Java preference is 120 also.   Maybe we should agree on a preference.  (It'll help with the review of non-space or non-formatting changes)